arangod/Indexes/SortedIndexAttributeMatcher.cpp

```diff
@@ -388,19 +417,11 @@ Index::SortCosts SortedIndexAttributeMatcher::supportsSortCondition(
       if (costs.coveredAttributes >= sortCondition->numAttributes()) {
         // sort is fully covered by index. no additional sort costs!
         // forward iteration does not have high costs
-        costs.estimatedCosts = itemsInIndex * 0.001;
-        if (idx->isPersistent() && sortCondition->isDescending()) {
-          // reverse iteration has higher costs than forward iteration
```

Comment:
Why was this removed? Is this statement no longer true?

Reply:
Added this back.